Taarafo.Core/Models/PostImpressions/PostImpression.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using Microsoft.EntityFrameworkCore;
+using Taarafo.Core.Models.Posts;
+using Taarafo.Core.Models.Profiles;
+
+namespace Taarafo.Core.Models.PostImpressions
+{
+    [Keyless]
```

Comment:
I would like to learn about this [Keyless]. Please share pointers.

Reply:
Here it is: https://docs.microsoft.com/en-us/ef/core/modeling/keyless-entity-types?tabs=data-annotations